### PR TITLE
Code Improvements

### PR DESCRIPTION
**Pull request by muizidn**
Better can be means:
 - Less code to write
 - Unified initializer
 - Statically typechecked

### CODE REVIEW

**Comment by muizidn** [comment]
@Shubham0812 Would you please review these? I would love to add more improvements. Thank you!

**Comment by Shubham0812** [comment]
Hey sure, I'll check it out when I get time. Thanks :D

**Comment by Shubham0812** [comment]
Hey, @muizidn it looks pretty good. There's a type in the protocol files, `clazzes` change it and then you can raise a PR :D
Well done.

File: Animatify/Extensions/UINibExtensions.swift [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+protocol UINibInstantiable {
+}
+extension UINibInstantiable {
+    static var nibName: String {
+        "\(Self.self)"
+    }
+}
+
+extension UINib {
+    convenience init(for clazz: UINibInstantiable.Type) {
```
[the remark this answers is not in the document]
Comment (muizidn) [reply Animatify/Extensions/UINibExtensions.swift:20]: I couldn't find `clazzes` 🤔

**Review by Shubham0812 — APPROVED** [review]

**Pull request merged**